src/frontends/onnx/tests/onnx_utils.hpp

```diff
@@ -6,4 +6,21 @@
 
 #include <string>
 
+#include "openvino/runtime/core.hpp"
+
 static const std::string ONNX_FE = "onnx";
+
+inline std::string backend_name_to_device(const std::string& backend_name) {
+    if (backend_name == "INTERPRETER")
+        return "TEMPLATE";
+    if (backend_name == "IE_CPU")
+        return "CPU";
+    if (backend_name == "IE_GPU")
+        return "GPU";
```

Comment:
Can we remove these legacy names and use device names?

Reply:
done

Reply:
I changed hardcoded valued to variables from ov::test::utils.
To get rid of INTERPRETER, IE_CPU and IE_GPU names, I need to change more deeply CMakeLists.txt and test manifest files.
So, I'd prefer to do it in scope of the next PR